strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/services/impl/ArtifactEntryServiceImpl.java

```diff
@@ -46,14 +46,16 @@
                                                  String repositoryId,
                                                  Map<String, String> coordinates)
     {
-        return findByCoordinates(storageId , repositoryId, coordinates, null, false);
+        return findByCoordinates(storageId , repositoryId, coordinates, 0, -1, null, false);
     }
 
     @Override
     @Transactional
     public List<ArtifactEntry> findByCoordinates(String storageId,
                                                  String repositoryId,
                                                  Map<String, String> coordinates,
+                                                 int skip,
+                                                 int limit,
```

Comment:
Consider adding some new class (`PagingCriteria` ? ) and one additional parameter here instead of two.

Reply:
yeah, I was thinking about this too. Will keep it in mind for the future.
thanks